lib/main.dart

```diff
@@ -1,113 +1,194 @@
+                          blurRadius: blurRadius,
+                          color: Colors.black,
+                          spreadRadius: spreadRadius,
+                        ),
+                      ],
+                    ),
+                    child: CircleAvatar(
+                      backgroundImage: AssetImage(
+                        "assets/images/me.jpg",
+                      ),
+                      radius: circleAvatarRadius,
+                    ),
+                  ),
+                  SizedBox(
+                    height: sizedBoxHeight,
+                  ),
+                  Text(
+                    'Horacio Duca',
+                    style: TextStyle(
+                      fontSize: titleFontSize,
+                      fontWeight: FontWeight.bold,
+                      color: Colors.black,
+                      shadows: [
+                        Shadow(
+                          blurRadius: blurRadius,
+                          color: Colors.black,
+                          offset: Offset(
+                            0.0,
+                            -2.0,
+                          ),
+                        ),
+                        Shadow(
+                          blurRadius: blurRadius,
+                          color: Colors.lightGreen,
+                          offset: Offset(
+                            5.0,
+                            -5.0,
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                  SizedBox(
+                    height: sizedBoxHeight,
+                  ),
+                  Text(
+                    'Flutter Developer Trainee',
+                    style: TextStyle(
+                      fontSize: subtitleFontSize,
+                      letterSpacing: letterSpacing,
+                      fontWeight: FontWeight.bold,
+                      color: Colors.black,
+                      shadows: [
+                        Shadow(
+                          blurRadius: blurRadius,
+                          color: Colors.black,
+                          offset: Offset(
+                            0.0,
+                            -2.0,
+                          ),
+                        ),
+                        Shadow(
+                          blurRadius: blurRadius,
+                          color: Colors.lightGreen,
+                          offset: Offset(
+                            5.0,
+                            -5.0,
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                  SizedBox(
+                    width: sizedBoxWidth,
+                    height: sizedBoxHeight,
+                    child: Divider(
+                      height: dividerHeight,
+                      color: Colors.black54,
+                    ),
+                  ),
```

Comment:
You don't need the SizedBox here, you can increase the dividerHeight adding the sizedBoxHeight, to keep the same height, and for the width use indent and endIndent.
You can use this as a guide: https://api.flutter.dev/flutter/material/Divider-class.html

Reply:
done